src/modules/cmd_duration.rs

```diff
@@ -0,0 +1,89 @@
+use crate::config::Config;
+use ansi_term::Color;
+
+use super::{Context, Module};
+
+/// Outputs the time it took the last command to execute
+///
+/// Will only print if last command took more than a certain amount of time to
+/// execute. Default is two seconds, but can be set by config option `min_time`.
+pub fn module<'a>(context: &'a Context) -> Option<Module<'a>> {
+    let mut module = context.new_module("cmd_duration")?;
+    let arguments = &context.arguments;
+    let elapsed = arguments
+        .value_of("cmd_duration")
+        .unwrap_or("invalid_time")
+        .parse::<u64>()
+        .ok()?;
+
+    let signed_config_min = module.config_value_i64("min time").unwrap_or(2);
```

Comment:
No spaces allowed in TOML keys 😛 

```suggestion
    let signed_config_min = module.config_value_i64("min_time").unwrap_or(2);
```